activities/vulnerability.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+##############################################################################
+# AUTHOR(S):    Qasim Adegbite
+##############################################################################
+import os
```

Comment:
Add empty line after `import os` (isortlinter reports 'Imports are incorrectly sorted and/or formatted')
```suggestion
import os

```